aten/src/ATen/cpu/vec/vec128/vec128_convert.h

```diff
@@ -17,7 +18,7 @@ inline void convertImpl(
     int64_t n) {
   uint64_t len = static_cast<uint64_t>(n);
   for (uint64_t i = 0; i < len; i++) {
-    dst[i] = static_cast<to_type>(src[i]);
+    dst[i] = c10::convert<dst_t>(src[i]);
```

Comment:
We'll be loosing some perf when doing convert rather than static cast. Do we really need/want to do something like that? casts outside of range are ill defined, we should simply not test for that behavior